tools/proofers/srchrep.php

```diff
@@ -57,35 +57,6 @@ function test_and_enable_nonregex()
     }
 }
 
-function show_regex_help()
```

Comment:
If this is getting removed from the popup, would it be a good idea to add a link to the s/r section of `prooffacehelp`?

Reply:
You mean like hide the regular expression help and hide behind a javascript link?

Reply:
If that means move all the help to `prooffacehelp` and just link to the help there, yes. :) -- which it looks like you've done already